mongo/options/autoencryptionoptions.go

```diff
@@ -93,6 +93,34 @@ func (a *AutoEncryptionOptions) SetBypassAutoEncryption(bypass bool) *AutoEncryp
 }
 
 // SetExtraOptions specifies a map of options to configure the mongocryptd process.
```

Comment:
Why are these options not just fields on `AutoEncryptionOptions`? Most of them seem to be publicly supported?

Reply:
Sounds like good reasoning to me; thanks for investigating.